pkg/build/builder/dockerutil.go

```diff
@@ -49,6 +49,26 @@ type DockerClient interface {
 	TagImage(name string, opts docker.TagImageOptions) error
 }
 
+func pullImage(client DockerClient, name string, authConfig docker.AuthConfiguration) error {
+	logProgress := func(s string) {
+		glog.V(0).Infof("%s", s)
```

Comment:
This function is what gets passed to the progress reporter to print out progress. But yes, it's being used.